fossunited/fossunited/user_utils.py

```
@@ -4,50 +4,141 @@
+    if doc.user_type == "Website User" and not doc.flags.get("skip_approval_flow"):
+        frappe.db.set_value("User", doc.name, "enabled", 0, update_modified=False)
+        # set_unique_username sets doc.username in memory only - persist it so
+        # approve_user / _create_profile receive the correct username later
+        frappe.db.set_value("User", doc.name, "username", doc.username, update_modified=False)
+
+        frappe.sendmail(
+            recipients=[doc.email],
+            subject="Your FOSS United account is pending approval",
+            message=f"""<p>Hi {frappe.utils.escape_html(doc.full_name)},</p>
+<p>Thank you for signing up on FOSS United! Your account is currently <strong>pending review</strong> by our team.</p>
+<p>We'll email you once approved. This usually takes 1–2 working days.</p>
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)